.travis.yml

```diff
@@ -63,19 +61,33 @@ jobs:
         - nbsite_cleandisthtml.py ./_build/html take_a_chance
         - cd ..
       deploy:
-        - provider: pages
+        - edge:
+            branch: v1.8.47
```

Comment:
For my own enlightenment, can you explain what this setting does?  It's very mysterious. :-)

Reply:
It downgrades some travis component because the most recent version is broken: https://github.com/travis-ci/travis-ci/issues/9312

Reply:
Ah; very appropriately mysterious, then.  Should probably open an issue somewhere to remind us to remove this setting once Travis fixes the issue, as it will no doubt eventually cause us problems.